osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPoint.cs

```diff
@@ -50,6 +50,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            Delay(-TransformStartTime); // So our follow points start their transforms from t=0 (the start of the map), rather from the lead in time (where the clock is right now).

```

Comment:
you *never know anything about your parent* so *never make assumptions*. as soon as you start thinking "our parent is in x state" you are doing it wrong.